Meditation/Meditation/Model/TabViews.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import Combine
+
+final class TabViews: ObservableObject {
```

Comment:
This should be rather just `Tabs`. Try to not call classes `Views` if they don't implement the `View` protocol.